modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/LayoutProvider/handlers/fieldDeletedHandler.es.js

```diff
@@ -15,69 +15,30 @@
 import {FormSupport} from 'dynamic-data-mapping-form-renderer';
 
 import * as fieldDeletedHandler from '../../../../../src/main/resources/META-INF/resources/js/components/LayoutProvider/handlers/fieldDeletedHandler.es';
-import RulesSupport from '../../../../../src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es';
 import mockPages from '../../../__mock__/mockPages.es';
 
 describe('LayoutProvider/handlers/fieldDeletedHandler', () => {
 	describe('handleFieldDeleted(state, event)', () => {
-		it('calls removeRow() when row is left with no fields after delete operation', () => {
+		it('calls removeEmptyRows() when row is left with no fields after delete operation', () => {
 			const event = {
-				columnIndex: 0,
-				pageIndex: 0,
-				rowIndex: 0,
+				activePage: 0,
+				fieldName: 'radio',
+				removeEmptyRows: true,
 			};
 			const state = {
 				pages: mockPages,
 				rules: [],
 			};
 
-			const removeRowSpy = jest.spyOn(FormSupport, 'removeRow');
+			const removeRowSpy = jest.spyOn(FormSupport, 'removeEmptyRows');
```

Comment:
@alinedoleron, please name the variable consistently, e.g. rename it to `removeEmptyRowsSpy`. Thanks.